### DISCUSSION TITLE
Importer Stuck on Configuration with no exception

### DISCUSSION THREAD

**Discussion by KasparRosin** [firefly-iii/firefly-iii#9513] [General]
I ran into a funny situation where my importer was stuck on configuration step, without displaying any of the fields nor buttons (except the previous step and restart).  I searched the discussions and errors without much luck.

Decided to create this discussion, incase somebody else runs into this stupid issue 😅 
**The solution was to create the main bank account right after registration.**

**Comment by JC5** [comment]
Thanks for letting me know! This feels like a bug: the importer should complain about this. I will check it out!
